src/Layout/NavBar/index.tsx

```diff
@@ -26,19 +26,19 @@ export const NavBar = () => {
 
     return <a
       href="/"
-      className="-mx-3 block rounded-lg px-3 py-2.5 text-base font-semibold leading-7 text-sky-100 hover:bg-slate-700"
+      className="-mx-3 block rounded-lg px-3 py-2.5 text-base font-semibold leading-7 text-sky-100 hover:bg-black"
       onClick={() => { logOut() }}
     >
       <img src={iclogo} className="h-8 w-8 inline p-0 mb-1 mr-2 align-middle" />
       Logout ({auth.identity.slice(0, 6)}...{auth.identity.slice(-4)})
     </a>
   }
 
-  return <header className="col-span-12 bg-slate-800 text-sky-100 rounded-b-lg">
+  return <header className="col-span-12 bg-[#0d0d0d] text-sky-100 rounded-b-lg">
```

Comment:
Same recommenation here: Name this color and then use it in the tailwind.config.js as a theme option.

As is the way of developers here is a tool to create a custom theme:
https://daisyui.com/theme-generator/